pkg/testmachinery/testflow/node/node.go

```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/gardener/test-infra/pkg/testmachinery/testdefinition"
 )
 
+var artifacts []argov1.Artifact
```

Comment:
global var will lead to unknown behavior as the controller is executed concurrently

Reply:
done